t/07-metrics/002-histograms_sighup.t

```diff
@@ -0,0 +1,186 @@
+# vim:set ft= ts=4 sts=4 sw=4 et fdm=marker:
+use strict;
+use lib '.';
+use t::TestWasmX;
+
+skip_no_hup();
+
+our $workers = 2;
+our $total = 0;
+
+workers($workers);
+if ($workers > 1) {
+    master_on();
+}
+
+no_shuffle();
+plan_tests(6);
+run_tests();
+
+__DATA__
+
+=== TEST 1: SIGHUP metrics - define metrics and record histograms
+--- valgrind
+--- load_nginx_modules: ngx_http_echo_module
+--- wasm_modules: hostcalls
+--- config eval
+qq{
+    location /t {
+        proxy_wasm hostcalls 'on_configure=define_and_record_histograms \
+                              metrics=c2,g2,h2';
+        echo ok;
+    }
+}
+--- grep_error_log eval: qr/histogram "\d+":( \d+: \d+;)+/
+--- grep_error_log_out eval
+$::total += $::workers;
+qr/histogram "\d+": 1: $::total; 4294967295: 0;/
+--- no_error_log
+[error]
+[crit]
+[emerg]
+[alert]
```

Comment:
@casimiro This also confuses me a little: are we able to define metrics even when no `metric{}` block and no slab was defined in the configuration?

Reply:
I see we have a default slab_size but it isn't taken care of at the module/directive level, rather down at the metrics level.